tests/testthat/test-retrieveData.R

```diff
@@ -2,6 +2,8 @@ globalassign <- function(...) {
   for (x in c(...)) assign(x, eval.parent(parse(text = x)), .GlobalEnv)
 }
 
+tempDir <- tempdir()
+setConfig(mainfolder = tempDir, .verbose = FALSE)
```

Comment:
```suggestion
setConfig(mainfolder = tempDir, .verbose = FALSE, .local = TRUE)
```
Das sollte hier nichts kaputt machen, oder?